core/src/main/java/org/hisp/dhis/android/core/arch/api/internal/HttpServiceClient.kt

```diff
@@ -44,11 +44,11 @@ import okhttp3.ResponseBody.Companion.toResponseBody
 import org.koin.core.annotation.Singleton
 
 @Singleton
-internal class KtorServiceClient(
-    private val client: HttpClient,
+class HttpServiceClient(
+    val client: HttpClient,
```

Comment:
These two properties are now exposed in the public API. The `client` property is ktor-specific. Couldn't they be hidden by marking them as internal or private?

Reply:
It seems Public-API inline function cannot access non-public-API internal variables.

Reply:
the publishedAPI annotation doesn't help here?